src/components/IFrame.js

```diff
@@ -30,19 +30,70 @@ function getNewDotURL(url) {
     }
 
     if (pathname === 'policy') {
-        const workspaceID = params.policyID || '';
-        const section = urlObj.hash.slice(1) || 'overview';
-
-        return `workspace/${workspaceID}/${section}`;
+        const workspaceID = params.policyID ?? '';
+        const section = urlObj.hash.slice(1) ?? '';
```

Comment:
There are 2 better ways here to simplify readability for default value
1. While declaring section value
```suggestion
        const section = urlObj.hash.slice(1) ?? 'overview';
```